doc/user-guides/secure-protect-connect-single-multi-cluster.md

```diff
@@ -415,90 +425,86 @@ For instance, if you have one cluster in North America and another in the EU, yo
 
-NOTE: While this section uses the `kuadrantctl` tool, this is not essential. You can also create and apply an `AuthPolicy`, `RateLimitPolicy`, and `HTTPRoute` by using the `oc` or `kubectl` commands.
-
-### Prerequisites
+**NOTE**: While this section uses the `kuadrantctl` tool, this is not essential. You can also create and apply an `AuthPolicy`, `RateLimitPolicy`, and `HTTPRoute` by using the `oc` or `kubectl` commands.
 
-- You have installed `kuadrantctl`. You can find a compatible binary and download it from the [kuadrantctl releases page](https://github.com/Kuadrant/kuadrantctl/releases/tag/v0.2.2).
-- You have the ability to distribute resources generated by `kuadrantctl` to multiple clusters, as though you are a platform engineer.
-
-
-### Step 1 - Deploy the toystore app
-
-To begin, deploy a new version of the `toystore` app to a developer namespace as follows:
+### Deploy the toystore app
 
+Create the namespace for your application. 
 ```bash
-kubectl apply -f https://raw.githubusercontent.com/Kuadrant/Kuadrant-operator/main/examples/toystore/toystore.yaml -n ${devNS}
-```
 
+kubectl create ns ${DEVELOPER_NS}
+```
 
-### Step 2 - Set up HTTPRoute and backend
-
-Copy at least one of the following example OAS to a local location:
-
-- [Sample OAS for rate limiting with API key](../../examples/oas-apikey.yaml)
-
-- [Sample OAS for rate limiting with OIDC](../../examples/oas-oidc.yaml)
-
-Set up some new environment variables as follows:
+Deploy a new version of the toystore app to your developer namespace as follows: 
 
 ```bash
-export oasPath=examples/oas-apikey.yaml
-# Ensure you still have these environment variables setup from the start of this guide:
-export rootDomain=example.com
-export gatewayNS=api-gateway
+kubectl apply -f https://raw.githubusercontent.com/Kuadrant/Kuadrant-operator/main/examples/toystore/toystore.yaml -n ${DEVELOPER_NS}
 ```
 
-### Step 3 - Use OAS to define your HTTPRoute rules
-
-You can generate Kuadrant and Gateway API resources directly from OAS documents by using an `x-kuadrant` extension.
-
-NOTE: For a more in-depth look at the OAS extension, see the [kuadrantctl documentation](https://docs.kuadrant.io/latest/kuadrantctl/).
-
-You will use `kuadrantctl` to generate your `HTTPRoute`.
-
-NOTE: The sample OAS has some placeholders for namespaces and domains. You will inject valid values into these placeholders based on your previous environment variables.
-
-Generate the resource from your OAS as follows, (`envsubst` will replace the placeholders):
+### Set up HTTPRoute and backend
 
+Create a HTTPRoute
 ```bash
-cat $oasPath | envsubst | kuadrantctl generate gatewayapi httproute --oas - | kubectl apply -f -
+
+kubectl apply -f - <<EOF
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: toystore
+  labels:
+    deployment: toystore
+    service: toystore
+spec:
+  parentRefs:
+  - name: ${gatewayName}
+    namespace: ${gatewayNS}
+  hostnames:
+  - "api.${rootDomain}"
```

Comment:
This looks wrong as it is using the old env vars

Reply:
oops copy and past error